### PR TITLE
This branch is to create Mobile Navigation Menu for Portfolio project

### PR DESCRIPTION
**Pull request by steveWDamesJr**
**Add files:**

- app.js : Add external JS script file and connect to markup
- eslintrc.json : Install CSS, HTML, and Js linters

**Refactor markup through pair programming to include Responsive nav:**

- click to section nav links
- logo
- hamburger menu
- dropdown on click functionality

Debuged linters

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -26,25 +26,30 @@
 </head>
 
 <body>
+  <header>
+    <nav class="navbar">
+      <a href="#homepage-section" class="nav-branding">My Logo</a>
+      <ul class="nav-menu">
+        <li class="nav-item">
+          <a href="#homepage-section" class="nav-link">Portfolio</a>
```
Comment (DammyShittu) [inline index.html]: It's really nice that all links are going to the appropriate sections. 👍 However, what do you think about changing the `href` attribute of `Portfolio` from `#homepage-section` to the section that contains the projects that you're showcasing?
That way, it'll be easier for anyone viewing your portfolio on small screens to quickly see your works. I hope this makes sense. 😁

File: js/app.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const hamburger = document.querySelector(".hamburger");
+const navMenu = document.querySelector(".nav-menu");
+const bar2 = document.querySelector(".bar2")
+
+hamburger.addEventListener("click", () => {
+  hamburger.classList.toggle("active");
+  navMenu.classList.toggle("active");
+})
+
+document.querySelectorAll(".nav-link").forEach(n => n.addEventListener("click", () => {
```
Comment (DammyShittu) [inline js/app.js]: - Kindly run the command to install Eslint so that errors can be caught and fixed. 😉

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @steveWDamesJr , 

Your portfolio looks extremely amazing. Good job so far! 💯 

However, there are some issues that you still need to work on to go to the next project but you are **ALMOST THERE**!

## REQUIRED CHANGES ♻️

- I can see that you added the `.eslintrc.json` file to check for JavaScript linter errors. That's super nice. However, kindly run the command to install it as well. Please, check the [Eslint section](https://github.com/microverseinc/linters-config/tree/master/html-css-js#eslint) of the linters repository to see the command to run. You can do this 💪 

_Please, check the other comments under the review._

## CHEERS AND HAPPY CODING! 🎉 🎉 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by leolpaz — APPROVED** [review]
Hi @steveWDamesJr , 

Your project is complete!
Congratulations! 🎉🎉 

Now there is nothing left to do but to merge it! 🏆 





Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**